assets/blocks/course-outline/module-block/edit.js

```diff
@@ -122,14 +139,11 @@ const EditModuleBlock = ( {
 						/>
 					</div>
 					<div className="wp-block-sensei-lms-course-outline-module__lessons-title">
-						<h3 className="wp-block-sensei-lms-course-outline__clean-heading">
+						<div className="wp-block-sensei-lms-course-outline__clean-heading">
 							{ __( 'Lessons', 'sensei-lms' ) }
-						</h3>
+						</div>
 					</div>
 					<InnerBlocks
-						template={ [
-							[ 'sensei-lms/course-outline-lesson', {} ],
-						] }
```

Comment:
Had to remove this for now as due to a Gutenberg bug, this template lesson block's inspector controls get added to the module's inspector panel each time a block style preview is rendered :|